index.js

```diff
@@ -383,9 +407,15 @@ module.exports = class Hypercore extends EventEmitter {
   }
 
   _onpeerupdate (added, peer) {
-    if (added) this.extensions.update(peer)
+  // if (added) this.extensions.update(peer)
```

Comment:
Is this a todo or something that should be removed?

Reply:
Good catch, thanks!